src/Main.java

```diff
@@ -1,5 +1,63 @@
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello, World!");
+        System.out.println("task 1");
+        for (int i = 0; i <= 10; i++) {
+            System.out.println(i);
+        }
+        System.out.println(" ");
+        System.out.println("task 2");
+        for (int i = 10; i >= 0; i--) {
+            System.out.println(i);
+        }
+        System.out.println(" ");
+        System.out.println("task 3");
+        for (int i = 0; i <= 17; i += 2) {
+            System.out.println(i);
+        }
+        System.out.println(" ");
+        System.out.println("task 4");
+        for (int i = 10; i >= -10; i--) {
+            System.out.println(i);
+        }
+        System.out.println(" ");
+        System.out.println("task 5");
+        for (int i = 1904; i <= 2096; i = i + 4) {
```

Comment:
```suggestion
        for (int year = 1904; year <= 2096; year = year + 4) {
```
лучше назвать year, чтобы было понятно, что это такое